pyriemann_qiskit/utils/math.py

```diff
@@ -2,6 +2,7 @@
 from typing_extensions import deprecated
 
 import numpy as np
+from qiskit.quantum_info import Pauli, SparsePauliOp
 from pyriemann.utils.covariance import normalize
```

Comment:
Imports must be sorted in alphabetic order:
```suggestion
from pyriemann.utils.covariance import normalize
from qiskit.quantum_info import Pauli, SparsePauliOp
```